framework/include/transfers/MultiAppGeneralFieldTransfer.h

```diff
@@ -320,15 +320,41 @@ class MultiAppGeneralFieldTransfer : public MultiAppConservativeTransfer
       InterpCaches & interp_caches,                   // for higher order elemental values
       InterpCaches & distance_caches);                // same but helps make origin point decisions
 
-  /// Remove potential value conflicts that did not materialize because another source was closer
-  void examineValueConflicts(
-      const VariableName var_name,
+  /**
+   * Remove potential value conflicts that did not materialize because another source was closer
+   * Several equidistant valid values were received, but they were not closest
+   * @param var_index the index of the variable of interest
+   * @param dofobject_to_valsvec a data structure mapping dofobjects to received values and
+   * distances (used for nodal-value-dof-only variables and constant monomials)
+   * @param distance_caches a cache holding the distances received (used for higher order elemental
+   * variables)
+   * @param conflicts_vec a vector holding all the potential received conflicts
+   */
+  void examineReceivedValueConflicts(
+      const unsigned int var_index,
+      const DofobjectToInterpValVec & dofobject_to_valsvec,
+      const InterpCaches & distance_caches,
+      std::vector<std::tuple<unsigned int, dof_id_type, Point, Real>> & conflicts_vec);
+
+  /**
+   * Remove potential value conflicts that did not materialize because another source was closer
+   * Several equidistant valid values were found when computing values to send, but they were not
+   * closest, another value got selected
+   * @param var_index the index of the variable of interest
+   * @param dofobject_to_valsvec a data structure mapping dofobjects to received values and
+   * distances (used for nodal-value-dof-only variables and constant monomials)
+   * @param distance_caches a cache holding the distances received (used for higher order elemental
+   * variables)
+   * @param conflicts_vec a vector holding all the potential local conflicts
+   */
+  void examineLocalValueConflicts(
+      const unsigned int var_index,
       const DofobjectToInterpValVec & dofobject_to_valsvec,
       const InterpCaches & distance_caches,
-      std::vector<std::tuple<unsigned int, dof_id_type, Point, Real>> conflicts_vec);
```

Comment:
This was an atrocity.  We were accidentally passing by value before, so discarding changes entirely?

Reply:
The more I think about it ... does this need to be a separate parameter, even?  We only ever call it with `this->_local_conflicts`, right?  Likewise with `this->_received_conflicts` in that version above?  And this is a non-const non-static member function so we already have access to those that way.

Reply:
I can remove it from the parameters. Initially I was trying to use the same routine both local and received conflicts but that didn’t last

Reply:
I'm fine with it as-is, if you don't want to change it.  Almost done with the rest of the review, and so far I haven't seen anything that actually *needs* a change.